app/models/katello/glue/elastic_search/content_view_puppet_environment.rb

```diff
@@ -5,26 +5,6 @@ module Glue::ElasticSearch::ContentViewPuppetEnvironment
     def self.included(base)
       base.send :include, Ext::IndexedModel
```

Comment:
ah!

you need to get rid of this line:

https://github.com/Katello/katello/blob/master/app/lib/actions/katello/content_view_puppet_environment/create.rb#L7

and anywhere else that calls disable_auto_reindex!  on a content view puppet environment.  Its usually just the create/destroy/update actions